### DISCUSSION TITLE
Calculate PDD and profile effciently with dsurround

### DISCUSSION THREAD

**Discussion by YapingQi** [nrc-cnrc/EGSnrc#936] [Questions]
Hello everyone, I am going to calculate PDD or profile in a water box, like 40cm *40cm * 35cm in EGS DOSXYZnrc, and confused by the input of "Record 5&6" (voxel settings) and Record SC2 (dsurround*). If I score the one column of 100 voxels (0.2cm *0.2cm *0.1cm) along the Z+ axis,  I should define this column at Record 5&6, and use dsurround(1) to (4) to expand it to the water box size, am I right?
If it is, attached is the code. Thank you for providing me with your kind and professional help.

[dosexyz_input.txt](https://github.com/nrc-cnrc/EGSnrc/files/9989166/dosexyz_input.txt)

**Comment by YapingQi** [comment]
[dosexyz_input_new.txt](https://github.com/nrc-cnrc/EGSnrc/files/9989200/dosexyz_input_new.txt)

**Reply by YapingQi** [reply]
39.8 cm should be half, 19.9cm.
but I got error when running that file: 
![1668162260(1)](https://user-images.githubusercontent.com/87401135/201320618-8f5dce02-c3f9-47ed-9dfe-cc59c0bcf5d1.png)

**Reply by rtownson** [reply]
This is not a valid input file, try using dosxyz_gui to fix it or start fresh. E.g. I see that you have provided a phase-space file, but the inputs are set to source 2 (parallel beam).

**Reply by YapingQi** [reply]
In the DOSXYZnrc manual, isource =2 is phase space source incident from any direction. Is there any new manual?

**Reply by rtownson** [reply]
My mistake, in your input file the source is being read in as source 0, not source 2. Try loading your input file in dosxyz_gui and you will see. I think this is because of the lines of 0's in the file.
